packs/data/feats.db/underwater-marauder.json

```diff
@@ -23,7 +23,23 @@
                 }
             ]
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "aquatic-combat:not-flat-footed"
+            },
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "aquatic-combat:no-penalty-bludgeoning"
+            },
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "aquatic-combat:no-penalty-slashing"
+            }
```

Comment:
These two would be better handled with modifier suppression. The Aquatic Combat effect wouldn't need its flat modifier REs altered except to add slugs.

Reply:
Good point, was trying to prep a session at the last minute and completely forgot we could suppress with AdjustModifier. I'll rework with that.